src/DPD-REACT/random_external_state.h

```diff
@@ -111,7 +111,7 @@ int es_rand(es_RNG_t &state_)
 LAMMPS_INLINE
 double es_drand(es_RNG_t &state_)
 {
-  return 1.0 * es_urand64(state_) / MAX_URAND64;
+  return static_cast<double>(es_urand64(state_) / MAX_URAND64);
```

Comment:
are you sure this should not be?
```
static_cast<double>(es_urand64(state_)) / MAX_URAND64;
```

Reply:
@akohlmey Good point, I tweaked the change a little, trying to fix this warning:

```
../random_external_state.h:114:52: warning: implicit conversion from 'const uint64_t' (aka 'const unsigned long') to 'double' changes value from 18446744073709551614 to 18446744073709551616 [-Wimplicit-const-int-float-conversion]
  return static_cast<double>(es_urand64(state_)) / MAX_URAND64;
                                                 ~ ^~~~~~~~~~~
  ```

Reply:
Done--thanks